ChessboardPrinter/Program.cs

```diff
@@ -1,17 +1,19 @@
-﻿for (int i = 8; i > 0; i--)
+﻿Console.WriteLine("enter broard dimension: ");
```

Comment:
```suggestion
﻿Console.WriteLine("Enter board dimension: ");
```
Ще ми домовлялися, що розмір дошки має бути не більше 20ти, відсутня перевірка, що вводить користувач